arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,837 @@
+    },
+	"Operation": {
+      "description": "Automation REST API operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.Automation",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Runbooks, Jobs etc.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string"
+            }
+          },
+		  "description": "Provider, Resource and Operation values"
+        }
+      }
+    },
+	"ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }	
+      },
+	  "description": "Error response of an operation failure"
+    },
+	"Statistics": {
+      "properties": {
+        "counterProperty": {
+          "type": "string",
+          "description": "Gets or sets the property value of the statistic."
+        },
+        "counterValue": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the value of the statistic."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the startTime of the statistic."
+        },
+        "endTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the endTime of the statistic."
+        },
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id."
+        }
+      },
+      "description": "Definition of the statistic."
+    },
+    "StatisticsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Statistics"
+          },
+          "description": "Gets or sets a list of statistics."
+        }
+      },
+      "description": "The response model for the list statistics operation."
+    },
+	"Usage": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id of the resource."
+        },
+        "name": {
+          "$ref": "#/definitions/UsageCounterName",
+          "description": "Gets or sets the usage counter name."
+        },
+        "unit": {
+          "type": "string",
+          "description": "Gets or sets the usage unit name."
```

Comment:
Leave a commentHi Vishrut,
 We have received exceptions for the below issues:
 [Azure Automation] validations (size, range, regexes) on properties and parameters missing  #1020
#1020
 [Azure Automation] Descriptions need to be corrected in all Azure Automation resources  #1021
#1021

Samer is aware of above issues. The two issues above will be revisited. These descriptions were inherited from Hyak specs and we won't be changing descriptions or validation regex at the moment